src/preprocess/lits_data_split.py

```diff
@@ -28,6 +28,7 @@ def main(args):
             writer = csv.writer(f)
             writer.writerow(['path', 'type'])
             for path in sorted(train_folds):
+                path = args.resampled_data_dir / 'training' / path.stem
```

Comment:
Is there any concern to use `path.stem` rather than `path.name`?

Reply:
I'm not sure which one is better. The codes used here aim to get the directory of the data directory path. To my knowledge, `path.stem` would return the final component without the suffix. That's why I use the `path.stem` instead of `path.name`.

Reply:
I think the last part of `path` here doesn't have the suffix because it's patient id. Maybe the variable name `path` is confused.